rst_files/aiyagari.rst

```diff
@@ -4,14 +4,12 @@
 
 .. highlight:: julia
 
-
-*******************************************************
+******************
```

Comment:
@Nosferican the reason I make these long are because Sphinx actually cares how long they are (i.e., if the text is longer than the `**********`, it'll throw an error for the title not being long enough).

Reply:
Exactly, just make them as long as needed

Good
```
This is a title
===============
```
Bad
```
This is a title
========================================
```
Best to be explicit than ambiguous.

Reply:
OK, put it in the style guide I guess

Reply:
We should have an internal style guide for the RST (the current style if for the Jupyter output)

Reply:
I think you can just add a section to the existing document. No need to make a whole new one (i.e., if you're contributing rST, make sure you...)